packages/lab/src/multiline-input/MultilineInput.tsx

```diff
@@ -0,0 +1,215 @@
+import { clsx } from "clsx";
+import {
+  ChangeEvent,
+  ComponentPropsWithoutRef,
+  FocusEvent,
+  forwardRef,
+  ReactNode,
+  Ref,
+  TextareaHTMLAttributes,
+  useState,
+} from "react";
+import { makePrefixer, StatusAdornment, useControlled } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+
+import multilineInputCss from "./MultilineInput.css";
+
+const withBaseName = makePrefixer("saltMultilineInput");
+
+export interface MultilineInputProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "defaultValue">,
+    Pick<
+      ComponentPropsWithoutRef<"input">,
+      "disabled" | "value" | "defaultValue" | "placeholder"
+    > {
+  /**
+   * The marker to use in an empty read only Input.
+   * Use `''` to disable this feature. Defaults to '—'.
+   */
+  emptyReadOnlyMarker?: string;
+  /**
+   * End adornment component
+   */
+  endAdornment?: ReactNode;
+  /**
+   * Styling variant with full border. Defaults to false
+   */
+  fullBorder?: boolean;
```

Comment:
I'm feeling conflicted about having this prop. We've normally tried to push this sort of customisation as patterns

Reply:
cc @bhoppers2008 thoughts?

Reply:
My preference would be prop. It'll be frequently used (perhaps more so with than without), and with the new multiline input, it's less a customisation and more an explicit design decision.